cosim/black-parrot-example/ps.cpp

```diff
@@ -52,6 +52,9 @@ inline unsigned long long get_counter_64(bp_zynq_pl *zpl, unsigned int addr) {
   } while (1);
 }
 
+// TODO: how to pass number of cores in argv/argstr?
```

Comment:
I would just do an #define for now, the ps code is tightly coupled with the build